app/models/request.rb

```diff
@@ -1,4 +1,20 @@
 class Request < ApplicationRecord
-  belongs_to :sender, class_name: 'User'
+  # callbacks
+  
+
+  # Validations
+  validates :sender_id, uniqueness: { scope: :receiver_id }
```

Comment:
We can safely remove these two validations for sender_id and receiver_id. Our `belongs_to :sender` and `belongs_to :receiver` associations already handle those validations for us :)

Reply:
@rvvergara, removing this breaks the test shown below. Also, could you please explain the cases where `belongs_to :sender` or `belongs_to :receiver` handle validation? A resource would be nice as well. Thanks.

https://github.com/mosaaleb/fakebook/blob/adef170f4b8fb067efdbf4e607515706ad3222b2/spec/models/request_spec.rb#L8-L12